pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicacmgkehubfeaturemembership/_http.log

```diff
@@ -754,270 +2304,661 @@ X-Xss-Protection: 0
+      }
+    },
+    "projects/${projectNumber}/locations/global/memberships/gkehubmembership-ebgjrzwod6ylp7jy3jnq": {
+      "state": {
+        "code": "ERROR",
+        "description": "GKE Cluster missing",
+        "updateTime": "2024-07-11T22:56:50.424336916Z"
+      }
+    }
+  },
+  "name": "projects/${projectId}/locations/global/features/configmanagement",
+  "resourceState": {
+    "state": "ACTIVE"
+  },
+  "spec": {},
+  "updateTime": "2024-04-01T12:34:56.123456Z"
 }
 
 ---
 
-GET https://gkehub.googleapis.com/v1beta1/projects/example-project-01/locations/global/memberships/gkehubmembership-basic-acm-${uniqueId}?alt=json
+GET https://gkehub.googleapis.com/v1beta1/projects/${projectId}/locations/global/memberships/gkehubmembership-basic-acm-${uniqueId}?alt=json
 Content-Type: application/json
 User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
 
 404 Not Found
-Content-Type: application/json
+Cache-Control: private
+Content-Type: application/json; charset=UTF-8
+Server: ESF
+Vary: Origin
+Vary: X-Origin
+Vary: Referer
+X-Content-Type-Options: nosniff
+X-Frame-Options: SAMEORIGIN
+X-Xss-Protection: 0
 
 {
-  "code": 5,
-  "details": [],
-  "message": "membership \"projects/example-project-01/locations/global/memberships/gkehubmembership-basic-acm-${uniqueId}\" not found"
+  "error": {
+    "code": 404,
+    "message": "Resource 'projects/${projectId}/locations/global/memberships/gkehubmembership-basic-acm-${uniqueId}' was not found",
+    "status": "NOT_FOUND"
+  }
 }
 
 ---
 
-POST https://gkehub.googleapis.com/v1beta1/projects/example-project-01/locations/global/memberships?alt=json&membershipId=gkehubmembership-basic-acm-${uniqueId}
+POST https://gkehub.googleapis.com/v1beta1/projects/${projectId}/locations/global/memberships?alt=json&membershipId=gkehubmembership-basic-acm-${uniqueId}
 Content-Type: application/json
 User-Agent: kcc/controller-manager DeclarativeClientLib/0.0.1
 
 {
   "authority": {
-    "issuer": "https://container.googleapis.com/v1/projects/example-project-01/locations/us-central1-a/clusters/containercluster-basic-acm-${uniqueId}"
+    "issuer": "https://container.googleapis.com/v1/projects/${projectId}/locations/us-central1-a/clusters/gke-basic-acm-${uniqueId}"
   },
   "description": "A sample GKE Hub membership",
   "endpoint": {
     "gkeCluster": {
-      "resourceLink": "//container.googleapis.com/projects/example-project-01/locations/us-central1-a/clusters/containercluster-basic-acm-${uniqueId}"
+      "resourceLink": "//container.googleapis.com/projects/${projectId}/zones/us-central1-a/clusters/gke-basic-acm-${uniqueId}"
     }
   },
   "labels": {
     "cnrm-test": "true",
     "managed-by-cnrm": "true"
   },
-  "name": "projects/example-project-01/locations/global/memberships/gkehubmembership-basic-acm-${uniqueId}"
+  "name": "projects/${projectId}/locations/global/memberships/gkehubmembership-basic-acm-${uniqueId}"
 }
 
 200 OK
+Cache-Control: private
```

Comment:
If I read the code correctly, this means you would need to improve the MockGCP GKEHub to include the metadata

Reply:
Sorry for the confusion, since it is an old PR, the metadata is handle by the LROs in the mock. You can look at the "head" of the http.log to verify.